docs/core/versions/index.md

```diff
@@ -27,6 +27,15 @@ With .NET Core 2.0, downloads show a single version number in their file name. T
 
 The use of a single version number makes it easier for users to know what version of the SDK to install on their dev machines, and what the corresponding version of the shared framework should be when time comes to provision a production environment. When downloading an SDK or runtime, the version number you see is going to be the same.
 
+### Version binding
```

Comment:
It looks like renamed the document from "Binding" to "Version Selection", but the term hasn't been updated everywhere.  Is this intentional?

Reply:
As I was looking at @dsplaisted 's comment, I noticed 

"The use of a single version..."

would this read better as 

"The use of the same version number" 
or
"Using the same version number"

Reply:
@dsplaisted Got it. I had thought you meant to update "binding" for SEO, not because we weren't using the term.

@KathleenDollard Let's handle that with the next PR, based on #6081  The SDK and Runtime versions aren't aligned, so that section will change a lot.